packages/components/organisms/f-header/src/components/Header.vue

```diff
@@ -237,7 +237,7 @@ html:global(.is-navInView) {
     }
 
     .c-header--highlightBg {
-        background-color: $color-primary;
+        background-color: $color-content-brand;
```

Comment:
I'd change this to `$color-support-brand` as `-content` usually is for text in the tokens.